packages/vue-prism-editor/src/Editor.ts

```diff
@@ -1,17 +1,18 @@
-import Vue, { VNode } from 'vue';
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { defineComponent, h } from 'vue';
 
 import './styles.css';
-const KEYCODE_ENTER = 13;
-const KEYCODE_TAB = 9;
-const KEYCODE_BACKSPACE = 8;
-const KEYCODE_Y = 89;
-const KEYCODE_Z = 90;
-const KEYCODE_M = 77;
-const KEYCODE_PARENS = 57;
-const KEYCODE_BRACKETS = 219;
-const KEYCODE_QUOTE = 222;
-const KEYCODE_BACK_QUOTE = 192;
-const KEYCODE_ESCAPE = 27;
+const KEYCODE_ENTER = 'Enter';
+const KEYCODE_TAB = 'Tab';
+const KEYCODE_BACKSPACE = 'Backspace';
+const KEYCODE_Y = 'y';
+const KEYCODE_Z = 'z';
+const KEYCODE_M = 'm';
+const KEYCODE_PARENS = '(';
+const KEYCODE_BRACKETS = '{';
+const KEYCODE_QUOTE = '"';
+const KEYCODE_BACK_QUOTE = '`';
+const KEYCODE_ESCAPE = 'Escape';
```

Comment:
these are not keycodes anymore what was the problem?

Reply:
`event.keyCode` is deprecated
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

and `event.key` seems compatible with IE9+ 
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

It has nothing to do with vue 3 and I should remove this change from this PR.

Reply:
i think its okay we can keep the changes

Reply:
<img width="378" alt="Screen Shot 2020-10-16 at 2 37 44 PM" src="https://user-images.githubusercontent.com/5592465/96301729-38bbcf00-0fbd-11eb-816a-780bd8acf47b.png">